boa/core/test.py

```diff
@@ -64,7 +64,8 @@ def get_metadata(yml, config, is_pyproject_recipe=False):
 
                 d = toml.load(fi)["tool"]["boa"]
         else:
-            d = ruamel.yaml.safe_load(fi)
+            loader = YAML(typ="safe")
```

Comment:
typ or type?

Reply:
It's `typ` https://sourceforge.net/p/ruamel-yaml/code/ci/default/tree/main.py#l57

Reply:
Oh...